tools/cell_census_builder/__main__.py

```diff
@@ -91,6 +91,24 @@ def main() -> int:
 
     return cc
 
+def prepare_file_system(soma_path: str, assets_path) -> None:
```

Comment:
I moved this code into a function to facilitate testing. In the test, the `setUp` will prepare the environment so at least the datasets folder will be pre-created.